modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_18_1/ObjectViewColumnUpgradeProcess.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.upgrade.v3_18_1;
+
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import java.util.Map;
+
+/**
+ * @author Julián Vela
+ */
+public class ObjectViewColumnUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				"select distinct(objectFieldName) as objectFieldName from " +
+					"ObjectViewColumn where  objectFieldName = 'dateCreated' " +
+						"or objectFieldName = 'dateModified'");
+			PreparedStatement preparedStatement2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update ObjectViewColumn set objectFieldName = ? where " +
+						"objectFieldName  = ?");
+			ResultSet resultSet = preparedStatement1.executeQuery()) {
```

Comment:
@jvelalr why do we need this first `select` which always returns the same two strings: `dateCreated` and `dateModified`? Why don't you simply do:

```
@Override
protected void doUpgrade() throws Exception {
	runSQL(
		"update ObjectViewColumn set objectFieldName = 'createDate' " +
			"where objectFieldName = 'dateCreated'");
	runSQL(
		"update ObjectViewColumn set objectFieldName = 'modifiedDate' " +
			"where objectFieldName = 'dateModified'");
}
 ```
 ?